page/docs/guide/run.md

```diff
@@ -42,6 +43,8 @@ optional arguments:
                         Specify the network to fork: can be a URL (e.g. https://alpha-mainnet.starknet.io) or network name (valid names: alpha-goerli, alpha-goerli2, alpha-mainnet)
   --fork-block FORK_BLOCK
                         Specify the block number where the --fork-network is forked; defaults to latest
+  --chain_id CHAIN_ID
+                        Specify the chain id as string, MAINNET or TESTNET
```

Comment:
If you change the help string format, don't forget to update this as well.

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/358/commits/96c4fef18c5b5e280a97e55eacef6f4fc4b00115